src/linagora.esn.calendar/app/event-view/external-user/event-view-external-user.controller.js

```diff
@@ -25,6 +25,7 @@ const _ = require('lodash');
     }
 
     function changeParticipation(partstat) {
+      $rootScope.$emit('UPDATE_ACTION_EXCAL', self.linksMapping[partstat]);
       self.userAsAttendee.partstat = partstat;
```

Comment:
```suggestion
      $rootScope.$emit('UPDATE_ACTION_EXCAL', self.linksMapping[partstat]);

      self.userAsAttendee.partstat = partstat;
```

And you should have a constant for `UPDATE_ACTION_EXCAL`.